scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      disallowSleep: 'Please do not use browser.sleep() in protractor files'
+      disallowSleep: 'Please do not use browser.sleep() in protractor files',
+      disallowThen: 'Please do not use .then(), consider async/await instead'
```

Comment:
@U8NWXD Can you please confirm whether this message is correct?

Reply:
Yes, async/await is the preferred replacement for .then()